### PR TITLE
ADD Notification Page Automatic issue labelers

### PR DESCRIPTION
**Pull request by ABHISHEK-PANDEY2**
Add Notification Page and automatic issue labeler 
## Description

A new page for notifications are added
## Related Issue

#809 
## Screenshots or GIF (In case of UI changes):
image

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by Sarfraz-droid — APPROVED** [review]

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - "help"
+good first issue:
+  - "good first issue"
+CI/CD:
+  - "CI/CD"
+dependency:
+  - "(dependency|dependencies|npm)"
+Urgent:
+  - "(Urgent|urgent|important)"
+borken:
```
Comment (Sarfraz-droid) [inline .github/labels.yml:19]: typo . change it to `broken`

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

File: .github/labels.yml (hunk 1 above)
Comment (ABHISHEK-PANDEY2) [reply .github/labels.yml:19]: ok

**Review by Sarfraz-droid — DISMISSED** [review]

**Review by Maahi10001 — APPROVED** [review]

File: src/components/Notification/styles.jsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  Notification: {
+    position: "relative",
+    boxSizing: "border-box",
+    display: "flex",
+    width: "100%",
+    alignItems: "center",
+    padding: "15px 20px",
+    gap: "20px",
+    fontSize: "1.2rem"
+    // direction: "column"
```
Comment (samyakshah3008) [inline src/components/Notification/styles.jsx:48]: Same as above comment

**Review by samyakshah3008 — COMMENTED** [review]
Solid work @ABHISHEK-PANDEY2 💪 
However, I have added a few comments, please check and let me know your inputs on those comments😄
Cheers!

File: src/components/Notification/index.jsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    {
+      name: "Google Summer of Code",
+      img: [OrgUser],
+      date: "25 March, 2022"
+    },
+    {
+      name: "Google Summer of Code",
+      img: [OrgUser],
+      date: "25 March, 2022"
+    }
```
[the remark this answers is not in the document]
Comment (Sarfraz-droid) [reply src/components/Notification/index.jsx:45]: @samyakshah3008 its just for testing right now. Will be updated once the backend is in development

[the remark this answers is not in the document]
Comment (samyakshah3008) [reply src/components/Notification/index.jsx:45]: Sounds cool, thanks for clearing @Sarfraz-droid 😄

File: src/components/Notification/NotificationBox.jsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    handleClose();
+  };
+  const handleDelete = id => {
+    notifications.filter(notification => id != notification.id);
+  };
+  return (
+    <>
+      <Card
+        className={classes.Notification}
+        style={{ backgroundColor: notification.isRead ? "#fff" : "#D9F1FC" }}
```
[the remark this answers is not in the document]
Comment (Sarfraz-droid) [reply src/components/Notification/NotificationBox.jsx:28]: @ABHISHEK-PANDEY2 can be used instead `@mui/material/colors`

**Comment by Sarfraz-droid** [comment]
Hi @samyakshah3008, Thank you for your input! Please note currently this is a dev PR in GSOC 23. So some of the code does include dummy data. But thankyou for your suggestions. I'll request the changes for the same.

**Review by Sarfraz-droid — APPROVED** [review]

**Comment by samyakshah3008** [comment]
> Hi @samyakshah3008, Thank you for your input! Please note currently this is a dev PR in GSOC 23. So some of the code does include dummy data. This will be updated and fixed in the next PRs.
> 
> But thankyou for your suggestions. I'll update the changes for the same.

Pleasure @Sarfraz-droid 😄 ✨  
Sure, I'll make sure about that dummy data point in next PR reviews! 

Cheers.

**Review by shivareddy6 — APPROVED** [review]

**Pull request merged**